### ISSUE TITLE
'jobject' must not be IntPtr.Zero

### ISSUE THREAD

**Issue by JoshuaNovak919** [rid00z/FreshMvvm#73]
After updating to `2.1.0` I am getting the following error every time I go back a page.
```
'jobject' must not be IntPtr.Zero.
Parameter name: jobject
```
Anyone else having this issue or know what may cause it?

**Comment by rid00z** [comment]
This would be be to do with Fresh more a Xamarin.Forms issue I would think.
We use it on 2.1 in many projects.

Let me know if it is fresh.



On Saturday, 21 May 2016, JoshuaNovak919 <notifications@github.com> wrote:

> After updating to 2.1.0 I am getting the following error every time I go
> back a page.
>
> 'jobject' must not be IntPtr.Zero.
> Parameter name: jobject
>
> Anyone else having this issue or know what may cause it?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/rid00z/FreshMvvm/issues/73>
>


-- 

*Michael Ridland | Technical Director | Xamarin MVP*

XAM Consulting - Mobile Technology Specialists

www.xam-consulting.com

Blog: www.michaelridland.com

**Comment by JoshuaNovak919** [comment]
@rid00z Hmm i'll have to dig in deeper. It didn't happen until I updated to `2.1.0`, so it's kind of odd. Did anything big change from `2.0.1` to `2.1.0`?

**Comment by rid00z** [comment]
Actually this could be us. If u have time just download the code and hook
up the project into your project. See if you can find the exact location of
issue.






On Saturday, 21 May 2016, JoshuaNovak919 <notifications@github.com> wrote:

> @rid00z <https://github.com/rid00z> Hmm i'll have to dig in deeper. It
> didn't happen until I updated to 2.1.0, so it's kind of odd. Did anything
> big change from 2.0.1 to 2.1.0?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/rid00z/FreshMvvm/issues/73#issuecomment-220740289>
>


-- 

*Michael Ridland | Technical Director | Xamarin MVP*

XAM Consulting - Mobile Technology Specialists

www.xam-consulting.com

Blog: www.michaelridland.com

**Comment by rid00z** [comment]
@JoshuaNovak919  Is this still a issue?

**Comment by JoshuaNovak919** [comment]
@rid00z Yes, but I haven't had a chance to narrow it down more. I'll try and knock it out tomorrow.
